Formula/spice-gtk.rb

```diff
@@ -62,6 +61,8 @@ class SpiceGtk < Formula
     sha256 "2b020ecf7d21b687f219b71ecad3631f644a47f01403fa1d1036b0c6416d70fb"
   end
 
+  patch :DATA
```

Comment:
Would be nicer to use the merged commit: https://gitlab.com/keycodemap/keycodemapdb/-/commit/c2c09b3f4c1d7688711e6501338ac88017d4341c

At minimum, we should link to the MR.